src/api/models/iuf/schemaValidator/schemas/iuf-manifest-schema.yaml

```diff
@@ -396,6 +396,17 @@ properties:
                     type: string
                 md5sum:
                   $ref: "#/$defs/MD5Sum"
+                arch:
+                  description: Target architecture for the recipe.
+                  example: aarch64
+                  enum:
+                    - aarch64
+                    - x86_64
+                  type: string
+                require_dkms:
```

Comment:
Is the dkms info required for ```images``` as well?

Reply:
I am pulling this from the `ims` api spec, but let me clear that up with David.

Reply:
Looks like he commented in a slack thread about this and that it should be included for this change.